projects/observability/src/shared/graphql/model/schema/filter/graphql-filter.ts

```diff
@@ -23,6 +23,5 @@ export const enum GraphQlOperatorType {
   Like = 'LIKE',
   In = 'IN',
   NotIn = 'NOT_IN',
-  ContainsKey = 'CONTAINS_KEY',
-  ContainsKeyValue = 'CONTAINS_KEY_VALUE'
+  ContainsKey = 'CONTAINS_KEY'
```

Comment:
Aw, was hoping we could get rid of this operator too. Maybe in the future.

Reply:
This is its own thing, it's distinct functionality unique to maps. It no longer takes an array though, so it's just like "Tags CONTAINS_KEY span.kind"